### DISCUSSION TITLE
Navidrome 0.51.1 and Artist biography

### DISCUSSION THREAD

**Discussion by ghost** [navidrome/navidrome#2938] [Q&A] [answered]
I have installed new 0.51.1 and the biography artist is not visible (it's blank), 
Also In my android client (Tempo) the "Radio Artist" feature not work anymore
In the previuos install work perfectly

Another question, but a web app for mobile is in wip?
thanks

**Comment by metalheim** [comment, marked as answer]
Version 0.51 had a breaking change about the last.fm Integration. Did you add your own api key like it is suggested in the release notes for 0.51? 

![Screenshot_20240406-071836](https://github.com/navidrome/navidrome/assets/44159957/9d9bf9d3-abde-4613-850d-c1d314882ea2)

**Comment by ghost** [comment]
Ok, thanks, now work

**Reply by metalheim** [reply]
glad it works now! Can you please mark the discussion as "answered"?
